pyconpune/templates/volunteer.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'site_base.html' %}
+    {% block body %}
```

Comment:
Don't indent django templating code because the final generated html will then have wrong indentation.